File: ejercicio_17/ejercicio_17.java
import java.util.Scanner;
public class ejercicio_17 {
    public static void main(String[] args) {
        int[] numero = new int[10];
        boolean creciente=false, decreciente=false;
        Scanner entrada = new Scanner(System.in);
        System.out.println("Ingresa numeros de 1 al 10:");
        for (int i = 0; i < 10; i++) {
            System.out.print("Ingresa el número "+ (i+1)+ " : ");
            numero[i] = entrada.nextInt();
        }
        for(int i = 0; i < 9; i++ ){
            if (numero[i]<numero[i+1]) {
                creciente = true;
            }
            if (numero[i]>numero[i+1]) {
                decreciente = true;
            }
        }
        if (creciente==true && decreciente==false){
            System.out.print("Los números que ingresaste estan en orden creciente ");
        }else if (decreciente==false && creciente==true) {
            System.out.print("Los números que ingresaste estan en orden decreciente ");
        }else if (decreciente==true && creciente==true) {
            System.out.print("Los números que ingresaste estan en desorden ");
        }else if (decreciente==false && creciente==false) {
            System.out.print("Los números que ingresaste todos son iguales ");
        }
        entrada.close(); 
    }
    
}

File: ejercicio_17/ejercicio_17.py
numero = [0] * 10
creciente = False
decreciente = False
print("Ingresa números del 1 al 10:")
for i in range(10):
    numero[i] = int(input(f"Ingrese el número {i + 1}: "))
for i in range(9):
    if numero[i] < numero[i + 1]:
        creciente = True
    if numero[i] > numero[i + 1]:
        decreciente = True
if creciente and not decreciente:
    print("Los números que ingresaste están en orden creciente ")
elif decreciente and not creciente:
    print("Los números que ingresaste están en orden decreciente ")
elif decreciente and creciente:
    print("Los números que ingresaste están en desorden ")
else:
    print("Los números que ingresaste todos son iguales ")